draft-ietf-quic-qpack.md

```diff
@@ -811,11 +819,13 @@ An encoder cannot infer from this instruction that any updates to the dynamic
 table have been received.
 
 
-## Request and Push Streams
+## Header Block Instructions
+
+HTTP/3 endpoints convert header lists to headers blocks and exchange them inside
+HEADERS and PUSH_PROMISE frames. Header block instructions reference the static
+table, or dynamic table in a particular state without modifying it.
```

Comment:
Same comment as before.  The primary purpose is to convey the header field values.  That they reference various tables to do so is secondary.

Reply:
Hmm. I think I agree with your position, which is why I have some struggle with the text as it currently stand. The changes here try to carry the existing QPACK position and text where possible. I'll take editor(s) cue on whether to broaden the changes in this PR.

AFAICT the QPACK approach was to discuss instructions and how they relate to the dynamic table. In that sense, this paragraph provides a clear statement that these instructions do not mutate.

Reply:
I'm OK with that aspect of this.  However, I would first say what these are for, then explain that they don't refer to the tables, but don't modify them.

Reply:
I'm OK with the text here.